src/plugins/point_cloud/PointCloud_TEST.cc

```diff
@@ -18,35 +18,156 @@
 #include <gtest/gtest.h>
 #include <string>
 
+#include <gz/msgs/pointcloud_packed.pb.h>
+#include <gz/msgs/PointCloudPackedUtils.hh>
+#include <gz/msgs/Utility.hh>
+
 #include <gz/common/Console.hh>
 #include <gz/common/Filesystem.hh>
+#include <gz/rendering/RenderEngine.hh>
+#include <gz/rendering/RenderingIface.hh>
+#include <gz/rendering/Scene.hh>
+#include <gz/transport/Node.hh>
 #include <gz/utils/ExtraTestMacros.hh>
 
 #include "test_config.hh"  // NOLINT(build/include)
 #include "gz/gui/Application.hh"
+#include "gz/gui/GuiEvents.hh"
```

Comment:
These headers aren't needed
```suggestion
#include <gz/transport/Node.hh>
#include <gz/utils/ExtraTestMacros.hh>

#include "test_config.hh"  // NOLINT(build/include)
#include "gz/gui/Application.hh"
```